src/posts/spotlights/jordan.md

```diff
@@ -10,18 +10,105 @@ mapsLink: https://www.google.com/maps/place/Jordan/@31.1675526,36.1454843,7.94z/
+
+### Jeep tour in Wadi Rum
+
+Touring the Wadi Rum desert is an sureal experience. You feel small as you climb on to a sand dune to get a 360 degree view of the surrounding area. There's something about the emptiness of the desert that cannot be described, it has to be felt in person.
+
+<figure>
+  <img src="/img/jordan/wadi-rum.jpg" alt="">
+  <figcaption>One of the many viewpoints in Wadi Rum.</figcaption>
+</figure>
+
+We spent around 2-3 hours touring the desert, with 4 stops for short hikes to viewpoints. Around 4pm we arrived at the bedouin camp, which would be our home for the next hours and for the night to come.
+
+<figure>
+  <img src="/img/jordan/wadi-rum-2.jpg" alt="">
+  <figcaption>Welcome to the red desert.</figcaption>
+</figure>
+
+### Bedouin camp experience
+
+There are dozens of camps around the desert that welcome guests from all over the world. We had a room ready at [Mohammed Mutlak Camp](http://www.booking.com/Share-5V1h7N) part of our trip, but you can also book on your own as most of the properties are on booking.com.
+
+<figure>
+  <img src="/img/jordan/wadi-rum-2.jpg" alt="">
+  <figcaption>Overlooking the camp.</figcaption>
+</figure>
+
+A room in a camp here is actually a wool tent with a hard floor, built and decorated in a traditional bedouin style. There's a big bed in the middle of the room with plenty of blankets for the cold night. There is also a power outlet and a single light source, but don't worry, you won't be staying in your room unless you're sleeping. The camp has a separate bigger building where you have shared toilets, running water and showers with hot water. The shared bathrooms are really clean and well maintained.
+
+Other than that, the camp area has an open fire where they serve tee non-stop and two bigger tents for eating and hanging around with the rest of the guests. One of the tents is enclosed and warmed. Dinner and breakfast were served by the staff of the camp and the food was delicious.
+
+<figure>
+  <img src="/img/jordan/wadi-rum-2.jpg" alt="Wadi Rum desert">
+  <figcaption>Sunset view.</figcaption>
+</figure>
+
+The best part is that there is no internet connection or phone signal, so you actually get to spend your time with the other guests. We had a great time getting to know our trip companions better and meeting other people from all over the world during the evening.
+
+Later during the night, you should definitely go outside and enjoy the starry night sky. There are few places on Earth where the light polution is so low.
+
+### Petra
+
+The second day of the trip was decicated entirely to Petra. All in all, we were at the site for about 6 hours, enough time to wander around and catch the best shots for your photo album. There are numerous hikes that you can take at Petra, some which require multiple hours to complete, so you can easily spend 1-2 full days there.
```

Comment:
dedicated
for about 6 hours. That's enough time to wander around and catch the best shots for your photo album. However, there are numerous hikes that you can take at Petra, some of which require multiple hours to complete, so you can easily spend 1-2 full days there if you have the time.